e2e/scripts/st_file_uploader.py

```diff
@@ -15,21 +15,18 @@
 import streamlit as st
 
 st.set_option("deprecation.showfileUploaderEncoding", True)
-result = st.file_uploader("Drop a file:", type=["txt"])
-# result = st.file_uploader("Drop a file:", type=["txt"], accept_multiple_files=False)
-if result is not None:
-    st.text(result.getvalue())
+single_file = st.file_uploader("Drop a file:", type=["txt"])
+if single_file is not None:
```

Comment:
`is not None` sounds like a double negative to me 😂 
I would prefer

```python
if single_file is None:
  ...
else:
  ...
```

but it's completely up to you.